WalletWasabi.Fluent/Views/Wallets/Send/TransactionPreviewView.axaml

```diff
@@ -67,73 +67,78 @@
                   <DockPanel>
                     <TextBlock Text="Suggestions to improve your privacy" HorizontalAlignment="Center" DockPanel.Dock="Top" />
 
-                    <ListBox DockPanel.Dock="Top" Background="Transparent" VerticalAlignment="Center"
-                             HorizontalAlignment="Center"
-                             Items="{Binding Suggestions}" SelectedItem="{Binding SelectedSuggestion}">
-                      <i:Interaction.Behaviors>
-                        <behaviors:ListBoxPreviewBehavior PreviewItem="{Binding PreviewSuggestion, Mode=TwoWay}" />
-                      </i:Interaction.Behaviors>
-                      <ListBox.Styles>
-                        <Style Selector="ListBoxItem">
-                          <Setter Property="Cursor" Value="Hand" />
-                        </Style>
-                        <Style Selector="ListBoxItem /template/ ContentPresenter#PART_ContentPresenter">
-                          <Setter Property="Background" Value="Transparent" />
-                        </Style>
+                    <Panel>
+                      <ListBox DockPanel.Dock="Top" Background="Transparent" VerticalAlignment="Center"
+                               HorizontalAlignment="Center"
+                               Items="{Binding Suggestions}" SelectedItem="{Binding SelectedSuggestion}"
+                               IsVisible="{Binding !IsBusy}">
```

Comment:
Remove the `IsVisible="{Binding !IsBusy}"` line as we want to see the suggestions as soon as the first one is available.